sdk/src/main/java/com/silanis/esl/api/model/BasePackage.java

```diff
@@ -88,6 +90,7 @@ public BasePackage ( ) {}
     protected java.util.Date _updated;
     protected String _visibility = "ACCOUNT";
     protected java.util.Date _created;
+    protected String _timezoneId;
```

Comment:
I can't understand why a package should have a timezone. Is not the time zone necessary for the user? Please link the Acceptance Criteria of the relevant ticket.

Reply:
Please see above comment for explanation.